src/EnergyPlus/DataSizing.hh

```diff
@@ -117,10 +117,34 @@ namespace DataSizing {
         Num
     };
 
-    // parameters for sizing
+    // parameters for sizing (keep this for now to avoid plant sizing output changes)
     constexpr int NonCoincident(1);
     constexpr int Coincident(2);
-    constexpr int Combination(3);
+
+    // parameters for zone and system sizing concurrence method
+    enum class SizingConcurrence
+    {
+        Invalid = -1,
+        NonCoincident,
+        Coincident,
+        Num
+    };
+
+    constexpr std::array<std::string_view, static_cast<int>(SizingConcurrence::Num)> SizingConcurrenceNamesUC{"NonCoincident", "Coincident"};
+
+    // parameters for coil sizing concurrence method
+    enum class CoilSizingConcurrence
+    {
+        Invalid = -1,
+        NonCoincident,
+        Coincident,
+        Combination,
+        NA,
+        Num
+    };
```

Comment:
You can probably combine these because Combination and NA is not a choice field in Sizing:Zone Type of Space Sum to Use.

Reply:
Yeah, I did that originally, but then I didn't like it. Do we have other examples of using just a subset of an enum?

Reply:
I think the best example is the fans. Not all fan types are allowed in all parents yet there is just one enum.

    fanCoil.fanType = static_cast<HVAC::FanType>(getEnumValue(HVAC::fanTypeNamesUC, Alphas(9)));

    enum class FanType
    {
        Invalid = -1,
        Constant,
        VAV,
        OnOff,
        Exhaust,
        ComponentModel,
        SystemModel,
        Num
    };

    constexpr std::array<std::string_view, (int)FanType::Num> fanTypeNames = {
        "Fan:ConstantVolume", "Fan:VariableVolume", "Fan:OnOff", "Fan:ZoneExhaust", "Fan:ComponentModel", "Fan:SystemModel"};

Reply:
Here's an option for working with a subset of an `enum`:

```cpp
constexpr std::array<bool, (int)FanType::Num> fanTypeInSubset1 = {true, false, true, true, false, false};
```

Reply:
> ```c++
> fanTypeInSubset1 
> ```

Yes, go on . . .

Reply:
> > ```c++
> > fanTypeInSubset1 
> > ```
> 
> Yes, go on . . .

Then you can use either of the two arrays or explicit comparisons to check whether the `enum` value you get is in Subset1 or not.